### DISCUSSION TITLE
Can I use this with Unity for real-time speech-to-text input?

### DISCUSSION THREAD

**Discussion by AntDX316** [deepgram/community#700] [General help] [answered]
Can I use this with Unity for real-time speech-to-text input?

If not, can you make a Unity Package?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment, marked as answer]
@AntDX316  Checkout this blog on using [Deepgram + Unity.](https://deepgram.com/learn/deepgram-unity-tutorial) it's a bit old but might be able to point you in the right direction for getting started.
